docs/utilities/data_classes.md

```diff
@@ -528,6 +529,23 @@ In this example, we also use the new Logger `correlation_id` and built-in `corre
         return { "markdown": f"# {echo}" }
     ```
 
+### CloudWatch Alarm State Change Action
+
+[CloudWatch supports Lambda as an alarm state change action](https://aws.amazon.com/about-aws/whats-new/2023/12/amazon-cloudwatch-alarms-lambda-change-action/){target="_blank"}.
+You can use the `CloudWathAlarmEvent` data class to access the fields containing such data as alarm information, current state, and previous state.
+
+=== "app.py"
+
+    ```python
+    from aws_lambda_powertools.utilities.data_classes import event_source, CloudWatchAlarmEvent
+
+    @event_source(data_class=CloudWatchAlarmEvent)
+    def lambda_handler(event: CloudWatchAlarmEvent, context):
+        if event.state.value.name == "ALARM":
+            print(f"{event.alarm_name} is on alarm because {event.state.reason}...")
+            do_something_with(event.alarm_arn)
+    ```
+
```

Comment:
Please move this example to `examples/event_sources`.

Reply:
Yes, you can remove the code from here and include the file like this https://github.com/aws-powertools/powertools-lambda-python/blob/develop/docs/utilities/data_classes.md?plain=1#L999

Reply:
Done, thanks!